phoenix-core/src/it/java/org/apache/phoenix/end2end/IndexToolIT.java

```diff
@@ -633,6 +635,83 @@ public void testSplitIndex() throws Exception {
         }
     }
 
+    /**
+     * Test that index tool fails when data table splits
+     */
+    @Test
+    public void testDataTableSplit() throws Exception {
```

Comment:
This test verifies that the "end region count" is calculated to be non-zero and fails the index tool when the index tool expects 0 regions. Seems like we'd also need a test that verifies that the initial count is populated into the M/R config properly.